diff --git a/node_modules/@greymass/eosio/lib/eosio-core.m.js b/node_modules/@greymass/eosio/lib/eosio-core.m.js
index 0da785d..a72e3db 100644
--- a/node_modules/@greymass/eosio/lib/eosio-core.m.js
+++ b/node_modules/@greymass/eosio/lib/eosio-core.m.js
@@ -3849,7 +3849,7 @@ class FetchProvider {
             // ignore json parse errors
         }
         const headers = {};
-        for (const [key, value] of response.headers.entries()) {
+        for (const [key, value] of Object.entries(response.headers)) {
             headers[key] = value;
         }
         return { headers, status: response.status, json, text };
